jung-visualization/src/main/java/edu/uci/ics/jung/visualization/subLayout/GraphCollapser.java

```diff
@@ -9,19 +9,22 @@
  */
 package edu.uci.ics.jung.visualization.subLayout;
 
-import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
 import com.google.common.graph.EndpointPair;
 import com.google.common.graph.MutableNetwork;
 import com.google.common.graph.Network;
 import com.google.common.graph.NetworkBuilder;
 import java.util.Collection;
+import java.util.List;
 import java.util.Set;
-import java.util.logging.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class GraphCollapser {
 
-  private static final Logger logger = Logger.getLogger(GraphCollapser.class.getClass().getName());
+  private static final Logger logger = LoggerFactory.getLogger(GraphCollapser.class);
```

Comment:
Again, I could be missing something, but I believe this logger isn't used either?